package.json

```diff
@@ -19,27 +20,25 @@
     "test-applications": "find ./test/applications -name '*.js' | xargs mocha --compilers js:babel-core/register",
     "test": "npm run test-base && npm run test-utilities",
     "test-experimental": "npm run test-recurrent-matrix && npm run test-recurrent && npm run test-utilities",
-    "dist": "babel src --out-dir dist --source-maps",
-    "browser": "browserify ./index.js -p licensify -o browser.js -s brain",
-    "browser-min": "browserify ./index.js -p licensify -s brain -g uglifyify -o browser.min.js",
-    "make": "rimraf dist && npm run dist && git add ./dist && npm run browser && npm run browser-min",
+    "bundle": "parcel build ./src/index.js --out-dir ./dist --out-file brain.js --no-minify",
+    "bundle-minified": "parcel build ./src/index.js --out-dir ./dist --out-file brain.min.js",
+    "dist": "npm run bundle && npm run bundle-minified",
     "clean": "rimraf node_modules"
   },
-  "main": "./index.js",
+  "main": "./src/index.js",
```

Comment:
Shouldn't this be from the dist folder?  If not, we'd be publishing es6 to an es5 world.

Reply:
Node.js version 6 and beyond supports es6/es-2015. So no need to worry. But we can still make sure we are not using any features from es7 (if that is necessary we can always include build for node.js). 

I will still test this script on older node.js versions.

Reply:
What about `import`?

Reply:
We can replace all import calls with require. (this will not effect browsers as we are using babel for that). 

This might feel over-burdening to you, I am suggesting to stick with this pattern because this way you can focus on writing scripts with only node.js specs in mind and script will be in-consistent with the node.js code writing pattern.

Reply:
Are there any other compatibility issues we'd need to think about if we remove imports?

Reply:
Too, doesn't the build create something that can be consumed by both browsers and node?

Reply:
I need to test extensively in node.js to make sure there is no other issue along with all the tests.  

Yes, it does. The build script can already be consumed in node.js.

Reply:
Forgot to mention that purpose of using `index` file from `src` directory was to enable somewhat easier debugging when using in node.js (multiple files will be required where breakpoints can be set inside individual files) where as dist folder has one single large file.

Reply:
We have got time to think about this. I will focus on fixing the tests for now.

Reply:
Very cool.